libcalico-go/config/crd/crd.projectcalico.org_bgpfilters.yaml

```diff
@@ -12,21 +11,27 @@ spec:
     listKind: BGPFilterList
     plural: bgpfilters
     singular: bgpfilter
+  preserveUnknownFields: false
```

Comment:
@caseydavenport Do we still need this `preserveUnknownFields` flag? Is this flag set to `false` for some historical reasons, like converting CRDs from `apiextensions.k8s.io/v1beta1` to `apiextensions.k8s.io/v1`?

Reply:
To be honest... this knowledge may be lost to time. let me see if I can dig anything up, but I believe it is some arcane historical reason that we had this.